components/unit/question-input.js

```diff
@@ -0,0 +1,16 @@
+export default function QuestionInput({ id, text, value }) {
```

Comment:
looks good, probably a good practice to add `PropTypes` to every component

Reply:
Ok, I'll add them in the next PR.